Sage/carbon
# test(select): add test that covers selecting the last item when a OptionGroupHeader is used - FE-5623

Tests covers a bug that was accidentally addressed in v114.14.0

fixes #5746

### Proposed behaviour

Add cypress test to cover a bug we inadvertently fixed in v114.14.0.

### Current behaviour

No test to cover this fix.

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

You should be able to navigate to the last select item using downarrows.
[codesandbox link](https://codesandbox.io/s/select-component-scroll-error-forked-lnwc69?file=/src/index.js)

New Cypress test is sufficient and passes.
